easybuild/easyconfigs/q/QuickPIC/QuickPIC-20210224-gompi-2021b.eb

```
@@ -0,0 +1,49 @@
+# There is unfortunately right now no proper release, thus the
+# commit-tag is being used. Does not build in parallel.
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MakeCp'
+
+name = 'QuickPIC'
+local_commit = '776fcff'
+version = '20210224'
+
+homepage = 'https://github.com/UCLA-Plasma-Simulation-Group/QuickPIC-OpenSource'
+description = """ QuickPIC is a 3D parallel (MPI & OpenMP Hybrid) Quasi-Static PIC code, 
+which is developed based on the framework UPIC. QuickPIC can efficiently simulate plasma 
+based accelerator problems. This is the UCLA Plasma Simulation Group's official open-source 
+repository for QuickPIC."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+toolchainopts = {'pic': True, 'usempi': True, 'cstd': 'c99'}
+
+source_urls = ['https://github.com/UCLA-Plasma-Simulation-Group/QuickPIC-OpenSource/archive']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = ['quickpic-makefile.patch']
```

One last thing (had overlooked that) is the name of the patch file. `quickpic` should definitely have the same capitalization as `name`. Furthermore, I don't know if this patch is version-specific, but maybe it should be something like:
```suggestion
patches = ['QuickPIC-20210224_fix_paths_in_makefile.patch']
```
?

---

And the file itself still has to be renamed then as well, of course.

---

Done. I never really know how to name the files, so thanks for your suggestion!